postgresql/values.yaml

```diff
@@ -64,7 +64,7 @@ persistence:
   ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
   ##   GKE, AWS & OpenStack)
   ##
-  storageClass: netapp-block-standard
+  storageClass: netapp-file-standard
```

Comment:
This is the only thing I want to get to the bottom of. I can take a poke at getting `netapp-block-standard` working again tomorrow.